addendum.dict

```diff
@@ -246,6 +252,7 @@ Melonsquashville	𐑥𐑧𐑤𐑩𐑯𐑕𐑒𐑢𐑪𐑖𐑝𐑦𐑤
 Meltzer	𐑥𐑧𐑤𐑑𐑕𐑼
 Menno	𐑥𐑧𐑯𐑴
 metrop	𐑥𐑩𐑑𐑮𐑪𐑐
+Microsoft 𐑥𐑲𐑒𐑮𐑴𐑕𐑭𐑓𐑑
```

Comment:
Should be 𐑥𐑲𐑒𐑮𐑴𐑕𐑪𐑓𐑑 to comply with principle 14 at https://www.shavian.info/spelling/